assets/airports/cyhz.json

```diff
@@ -0,0 +1,999 @@
+            },
+            "body": ["MUVOX"],
+            "rwy": {
+                "05": [["ODKAS", "A30|S200-"], "#053"],
+                "14": [["TETAR", "A30|S200-"], "#143"],
+                "23": [["LOGPO", "A50|S210-"], ["DUTIR", "A40|S210-"], "#053"],
+                "32": [["EBLUV", "A50|S210-"], ["EMBAX", "A40|S210-"], "#143"]
+            },
+            "draw": [
+                ["EBONY*", "EBKID", "MUVOX", "TETAR"],
+                ["ALLEX*", "BAFDY", "MUVOX", "EBLUV", "EMBAX"],
+                ["MUVOX", "ODKAS"],
+                ["MUVOX", "LOGPO", "DUTIR", "LEROS"]
+            ]
+        },
+        "LIRLA2": {
+            "icao": "LIRLA2",
+            "name": "Lirla Two",
+            "entryPoints": {
+                "AGMIR": ["AGMIR"]
+            },
+            "body": ["LIRLA"],
+            "rwy": {
+                "05": [["PEPTA", "A50|S210-"], ["OBNOM", "A40|S210-"], "#233"],
+                "14": [["VIMLA", "A50|S210-"], ["IGVOP", "A40|S210-"], "#323"],
+                "23": [["LEROS", "A30|S200-"], "#233"],
+                "32": [["VOKIL", "A30|S200-"], "#323"]
+            },
+            "draw": [
+                ["AGMIR", "LIRLA", "VIMLA", "IGVOP"],
+                ["LIRLA", "VOKIL"],
+                ["LIRLA", "PEPTA", "OBNOM"],
+                ["LIRLA", "LEROS"]
+            ]
+        },
+        "PEGGY2": {
+            "icao": "PEGGY2",
+            "name": "Peggy Two",
+            "entryPoints": {
+                "TUSKY": ["TUSKY"],
+                "ELERI": ["ELERI"]
+            },
+            "body": ["ETGAR"],
+            "rwy": {
+                "05": [["ODKAS", "A30|S200-"], "#053"],
+                "14": [["ILOTA", "A50|S210-"], ["MODIR", "A40|S210-"], "DUNUB", "#323"],
+                "23": ["OBNOM", ["RUDBU", "A50|S210-"], ["OVTUV", "A40|S210-"], "#053"],
+                "32": [["NOTUB", "A50|S230-"], ["EMBAX", "A40|S210-"], "EBLIT", "#143"]
+            },
+            "draw": [
+                ["TUSKY", "ETGAR", "ILOTA", "MODIR", "DUNUB"],
+                ["ETGAR", "NOTUB", "EMBAX", "EBLIT"],
+                ["ETGAR", "ODKAS"],
+                ["ETGAR", "OBNOM", "RUDBU", "OVTUV"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "CYHZ",
+            "destination": "",
+            "category": "departure",
+            "route": "CYHZ.CYHZ4.CAAPE",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.792,
+            "airlines": [
+                ["aca", 29],
+                ["aal", 1],
+                ["fpo", 2],
+                ["cfg", 2],
+                ["dal", 1],
+                ["fle", 2],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wja", 15],
+                ["wen", 5],
+                ["fdx", 1],
+                ["kal", 2],
+                ["qtr", 2],
+                ["n/n/cessna", 8],
```

Comment:
I believe this should just be `n/cessna`? You'll be getting errors if what you have here is wrong.